Engines/Wine/Verbs/D9VK/script.js

```diff
@@ -28,6 +28,16 @@ class D9VK {
         return this;
     }
 
+    selectGithubVersion(wizard) {
```

Comment:
`wizard` is purely for cosmetic reason here. It is not really required and could be replaced by `this.wine.wizard()`

Reply:
We should decide how we handle these cases in general. In my opinion, we should not pass parameters which are not required.